pkg/helm/run.go

```diff
@@ -124,15 +122,15 @@ func Run(flags *hoflags.HelmOperatorFlags) error {
 	}
 
 	// Generates operator specific metrics based on the GVKs.
-	// It serves those metrics on "http://metricsHost:operatorMetricsPort".
-	err = kubemetrics.GenerateAndServeCRMetrics(cfg, []string{namespace}, gvks, metricsHost, operatorMetricsPort)
+	// It serves those metrics on "http://metricsHost:CRPortNumber".
+	err = kubemetrics.GenerateAndServeCRMetrics(cfg, []string{namespace}, gvks, metricsHost, metrics.CRPortNumber)
 	if err != nil {
 		log.Info("Could not generate and serve custom resource metrics: ", err.Error())
 	}
 
 	servicePorts := []v1.ServicePort{
-		{Port: operatorMetricsPort, Name: metrics.CRPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: operatorMetricsPort}},
-		{Port: metricsPort, Name: metrics.OperatorPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: metricsPort}},
+		{Port: metrics.CRPortNumber, Name: metrics.CRPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: metrics.CRPortNumber}},
+		{Port: metrics.PortNumber, Name: metrics.OperatorPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: metrics.PortNumber}},
```

Comment:
Nit: doesn't really matter, but you could also use `intstr.FromInt()` instead of declaring the full `IntOrString` struct.